drivers/bridge/bridge.go

```diff
@@ -98,6 +98,7 @@ type bridgeNetwork struct {
 
 type driver struct {
 	config      *configuration
+	configured  bool
```

Comment:
@mavenugo I don't understand what you mean. The current code doesn't introduce any change in behavior from caller perspective and still retains the functionality exactly as before. It just fixes the panic if `CreateNetwork` is called without a prior `Config` call and that's it.